packages/spindle-ui/src/InlineNotification/InlineNotification.css

```
@@ -0,0 +1,218 @@
+.spui-InlineNotification--inset {
+  border-radius: 8px;
+}
+.spui-InlineNotification--full {
+  width: 100vw;
+}
+.spui-InlineNotification-content {
+  align-items: center;
+  display: flex;
+  padding: 8px 12px;
+}
+.spui-InlineNotification-left {
+  margin-right: 8px;
+}
+.spui-InlineNotification-icon {
+  align-items: center;
+  display: flex;
+  font-size: 1.375rem;
+  justify-content: center;
+  padding-bottom: 5px;
+  padding-top: 5px;
```

ショートハンド警察👮
```suggestion
  padding: 5px 0;
```
（他の箇所も同様）